ert3/evaluator/poly.py

```diff
@@ -0,0 +1,8 @@
+import json
+
+def polynomial(a, b, c):
+    x_range = tuple(range(10))
+    result =  tuple(a * x**2 + b * x + c for x in x_range)
+    with open("output_function.json", "w") as f:
```

Comment:
Why does the function write to a file?

Reply:
Just for the sake of using the mechanism that loads results from files in storage. I can move the file writing outside the function. What do you think ?

Reply:
I think that would be better! If we need to persist something to a file as a temporary thing to make the wheels turn that should be hidden from our users and a plan (and issue) should be created for how to deal with this :)